plugins/DynamicsProcessor/DynamicsProcessor.h

```diff
@@ -65,6 +65,7 @@ class DynProcEffect : public Effect
 	bool m_needsUpdate;
 	
 	RmsHelper * m_rms [2];
+	double m_smoothRms[2];
```

Comment:
We should be using `std::array` rather than C-style arrays.
```suggestion
	std::array<double, 2> m_smoothRms;
```